### ISSUE TITLE
Accessibility: Screen reader cannot access IntelliSense dropdown list

### ISSUE THREAD

**Issue by chrisdias** [Microsoft/vscode#3640]

**Comment by alexandrudima** [comment]
There is a working version using aria alerts. For the proper implementation using aria concepts, let's track that work in #3787
